src/Builder.ts

```diff
@@ -48,7 +48,7 @@ export class Builder {
 
             /* istanbul ignore else */
             if (typeof processorInstance.preProcess === 'function') {
-                data = processorInstance.preProcess(fixture.name, data);
+                data = await Promise.resolve(processorInstance.preProcess(fixture.name, data));
```

Comment:
Promise.resolve is not needed. Will work and so

Reply:
For example 
```typescript
await processorInstance.preProcess(fixture.name, data);
```

Reply:
resolved